src/transport_picoquic.cpp

```diff
@@ -1,27 +1,48 @@
 #include "transport_picoquic.h"
 
+// PicoQuic.
+#include <picoquic.h>
 #include <picoquic_internal.h>
+#include <picoquic_packet_loop.h>
 #include <autoqlog.h>
 #include <picoquic_utils.h>
 #include <spdlog/spdlog.h>
-
+#include <picoquic_config.h>
+#include <picosocks.h>
+
+// Transport.
+#include "transport/transport.h"
+#include "transport/transport_metrics.h"
+#include "transport/safe_queue.h"
+#include "transport/stream_buffer.h"
+#include "transport/priority_queue.h"
+#include "transport/span.h"
+#include "transport/time_queue.h"
+#include <spdlog/logger.h>
+
+// System.
 #include <arpa/inet.h>
+#include <sys/socket.h>
 #include <cassert>
 #include <cstring>
 #include <ctime>
 #include <iostream>
 #include <thread>
-#include <unistd.h>
 #include <sstream>
+#include <cstdint>
+#include <optional>
+#include <utility>
+#include <memory>
+#include <mutex>
+#include <vector>
+#include <string>
+#include <chrono>
+#include <sstream>
+#include <stdexcept>
```

Comment:
Also, personally (though I'm sure I got this habit from somewhere vaguely official) I like to order the headers as
```cpp
#include "local/header.h"

#include <dependency/header.h>

#include <system/header>
```

That way your dependency headers are always explicitly after your locally declared headers so that you don't run into potential include errors where you didn't include a dependency header in a local header, but that gets hidden when the dependency header comes before the local header in this implementation file. Of course IWYU fixes that issue.